src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import cn from 'classnames';
```

Comment:
don't cut variable names
```suggestion
import classnames from 'classnames';
```